dev-embedded/openocd/openocd-0.10.0-r1.ebuild

```diff
@@ -36,9 +35,11 @@ RDEPEND="
 	)
 	ftdi? ( dev-embedded/libftdi:= )"
 
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
+DEPEND="${RDEPEND}"
 [[ ${PV} == "9999" ]] && DEPEND+=" >=sys-apps/texinfo-5" #549946
```

Comment:
texinfo is a `BDEPEND` (even if this clause isnt being triggered)

Reply:
Fixed